ansible/roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# tasks file for etcd
+
+- name: install etcd from centos extras repository
+  yum: name=etcd state=installed
```

Comment:
updated to `state=latest`. Not sure how will this be when there is a major version change with features breaking backward compatibility.